From bdc7332719d114557a4a14ff655549f1b447ee8c Mon Sep 17 00:00:00 2001
From: Artur Wiebe <artur@4wiebe.de>
Date: Fri, 14 Apr 2023 10:16:37 +0200
Subject: [PATCH] symlink to Cargo.lock

---
 Cargo.lock | 1 +
 1 file changed, 1 insertion(+)
 create mode 120000 Cargo.lock

diff --git a/Cargo.lock b/Cargo.lock
new file mode 120000
index 0000000..a048b8f
--- /dev/null
+++ b/Cargo.lock
@@ -0,0 +1 @@
+src/rust/Cargo.lock
\ No newline at end of file
-- 
2.40.0

